### ISSUE TITLE
What are the differences with combobulate

### ISSUE THREAD

**Issue by DamienCassou** [casouri/expreg#2]
[combobulate](https://github.com/mickeynp/combobulate) (from @mickeynp) offers tree-sitter based features, including expanding region. Do you know of any difference with your package?

One difference I see is that combobulate requires language-specific support to specify what kinds of tree-sitter nodes make sense from a user point of view (at least this is how I understand combobulate). On the contrary, expreg seems to be completely language agnostic.

**Comment by casouri** [comment]
Nah, just use combobulate if you already have it. This is mainly something I write for myself and some others happen to want to use too.

**Comment by mickeynp** [comment]
I never used the original expand region package, so I can't really say if it had more advanced features that this package also tries to mirror. I'm sure if you want expand region + ts only, then this is probably a better choice?
